packages/editor/src/components/visual-editor/style.scss

```diff
@@ -1,6 +1,6 @@
 .editor-visual-editor {
 	position: relative;
-	display: block;
+	display: flex;
 	background-color: $gray-300;
 
 	// Centralize the editor horizontally (flex-direction is column).
```

Comment:
A belated note: adding the `display: flex` was all that was needed thanks to the existing (though prior to this change ineffectual) `align-items: center` rule that is just below the comment. It would have been good to revise the comment since it’s incorrect. Also, personally, I think it’d have been nice to put the `display: flex;` and `align-items: center;` on adjacent lines.